inception-external-search-solr/pom.xml

```diff
@@ -0,0 +1,114 @@
+<!--
+  Copyright 2020
+  Ubiquitous Knowledge Processing (UKP) Lab
+  Technische Universität Darmstadt
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<!-- TODO : Checkstyle-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>de.tudarmstadt.ukp.inception.app</groupId>
+    <artifactId>inception-app</artifactId>
+    <version>0.16.0-SNAPSHOT</version>
```

Comment:
Should be 0.16.2

Reply:
I'm working on the Solr module. First thanks for the feedback. It helps me alot to learn good practices. I tried to modify the snapshot version to 0.16.2 but I have some issues with webanno dependancies. So I tested to use 0.17.0 and the projet build correctly. Do you prefer 0.16.2 or 0.17.0 ?

Reply:
0.17.0-SNAPSHOT is fine as well. Just merge in master.

Reply:
Please mind to update the target branch of the PR to master as well.

Reply:
did that